lib/nostr.js

```diff
@@ -201,3 +201,13 @@ class NDKNip46SignerURLPatch extends NDKNip46Signer {
     return super.connectionTokenInit(connectionToken)
   }
 }
+
+export function getNostrProfile (pubkey) {
+  try {
+    const { data } = nip19.decode(pubkey)
```

Comment:
We store all nostr pubkeys in hex so you don't need to do this + throws an error when it encounters hex.

Reply:
It errors out requireing when we don't decode first. 
<img width="743" alt="Screenshot 2025-04-16 at 04 34 17" src="https://github.com/user-attachments/assets/1beab03a-6aea-4e24-888f-9b99d21bfe92" />